Match.cpp

```diff
@@ -331,21 +331,115 @@ void Match::print( ostream& fout )
 		fout.write( (char*) &hom[1] , sizeof( bool ) );
 	} else
 	{
-		fout << node[0]->getID() << '\t';
-		fout << node[1]->getID() << '\t';
-		fout << ALL_SNPS.getSNP(snp_start).getChr() << '\t';
-		fout << ALL_SNPS.getSNP(snp_start).getPhysPos() << ' ';
-		fout << ALL_SNPS.getSNP(snp_end).getPhysPos() << '\t';
-		fout << ALL_SNPS.getSNP(snp_start).getSNPID() << ' ';
-		fout << ALL_SNPS.getSNP(snp_end).getSNPID() << '\t';
-		fout << ( snp_end - snp_start + 1) << '\t';
-		fout << setiosflags(ios::fixed) << setprecision(2) << distance << '\t';
-		if ( genetic ) fout << "cM" << '\t'; else fout << "MB" << '\t';
-		fout << dif;
-		for ( int n = 0 ; n < 2 ; n++ )
-			if ( hom[n] ) fout << '\t' << 1; else fout << '\t' << 0;
-		fout << endl;
+		if ( ALL_SAMPLES.useEmbarkRFGermlineOutput ) {
+			int key1 = stoi(node[0]->single_id);
+			int key2 = stoi(node[1]->single_id);
```

Comment:
Is it worth handling case where this can't convert? I don't think so, because that means something is truly wrong? i.e., something was passed in that wasn't really a proxy key?